File: src/main/java/com/shreyash/BankApplication/DTO/AccountInfo.java
package com.shreyash.BankApplication.DTO;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.math.BigDecimal;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class AccountInfo {
    private String accountHolderName;
    private BigDecimal accountBalance;
    private String accountNumber;
}

File: src/main/java/com/shreyash/BankApplication/dto/AccountInfo.kt
package com.shreyash.BankApplication.dto;

import com.shreyash.BankApplication.authorities.AccountType
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.Getter
import lombok.NoArgsConstructor;
import lombok.RequiredArgsConstructor
import lombok.Setter

import java.math.BigDecimal;

@Data
@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@RequiredArgsConstructor
data class AccountInfo(
    val customerID: String,
    val accountHolderName: String,
    val accountBalance: BigDecimal,
    val accountNumber: String,
    val accountType: AccountType
)
